apps/web/src/App.vue

```diff
@@ -2,17 +2,17 @@
   import useUsers from '@/composables/users'
   import { RouterView } from 'vue-router'
   import DefaultLayout from '@/layouts/default-layout.vue'
-  import { onMounted } from 'vue'
+  // import { onMounted } from 'vue'
```

Comment:
Can we just delete all commented out code? Still need `user` from `useUsers` here?

Reply:
Will do!